pkg/internal/controller/metrics/metrics.go

```diff
@@ -29,6 +29,15 @@ var (
 		Help: "Length of reconcile queue per controller",
 	}, []string{"controller"})
 
+	// ReconcileTotal is a prometheus counter metrics which holds the total
+	// number of reconciles per controller. It has two labels. controller label refers
+	// to the controller name and result label refers to the reconcile result i.e
+	// success, error, requeue, requeue_after.
+	ReconcileTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "controller_runtime_reconcile_total",
+		Help: "Total number of reconciles per controller",
```

Comment:
nit: reconcile is a verb, i.e. s/reconciles/reconciliations

Reply:
Addressed.